scripts/_events.js

```diff
@@ -103,24 +106,33 @@ function listenForIbcChannelEvents (network, source, dispatcher) {
           -------------------------------------------
           🧾 TxHash: ${txHash}
           🔍 Explorer URL: ${url}
-          -------------------------------------------\n`
-        );
-
+          -------------------------------------------\n`);
+        }
         dispatcher.removeAllListeners();
     });
 }
 
-function listenForIbcPacketEvents (network, dispatcher) {
+function filterPacketEvents(portAddress, network) {
```

Comment:
@kenobijon your original filtering did not include universal packets. there's not difference in the event from IBC POV so we need to check both of the config fields (for now)